lib/customizer.php

```diff
@@ -74,6 +74,21 @@ function gutenberg_customize_register( $wp_customize ) {
 }
 add_action( 'customize_register', 'gutenberg_customize_register' );
 
+/**
+ * Removes the core 'Widgets' panel from the Customizer if block based widgets are enabled.
```

Comment:
> if block based widgets are enabled

Seems we need to add this condition inside the function?

Reply:
😬